### PR TITLE
fix: avoid using filter option with `null` name

### PR DESCRIPTION
**Pull request by briacp**
## Which ticket is resolved?

- Default filters settings don't initialize options
  * https://sourceforge.net/p/omegat/bugs/1167/

### CODE REVIEW

**Review by miurahr — APPROVED** [review]

File: test/src/org/omegat/filters2/FilterMasterTest.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
+package org.omegat.filters2;
```
Comment (miurahr) [inline test/src/org/omegat/filters2/FilterMasterTest.java:1]: We need a copyright header here.

**Comment by miurahr** [comment]
New test causes  a compile error of test code in master branch. @briacp  could you check it?

**Comment by miurahr** [comment]
```
@@ -61,7 +61,7 @@ public class FilterMasterTest {
         tempFilter = Files.createTempFile(FilterMaster.FILE_FILTERS, null).toFile();
 
         FilterMaster.setFilterClasses(
-                Arrays.asList(new Class[] { org.omegat.filters3.xml.xhtml.XHTMLFilter.class }));
+                Arrays.asList(new Class<?>[] { org.omegat.filters3.xml.xhtml.XHTMLFilter.class }));
     }

```

**Comment by kosivantsov** [comment]
Could this one also be backported to `releases/5.8` and `releases/6.0`? Or [this comment](https://sourceforge.net/p/omegat/bugs/1167/#d162/b924) still applies?

**Pull request merged**